server/api/schema.ts

```diff
@@ -6,6 +6,7 @@ const typeDefs = gql`
         quantity: Int
         clientName: String
         requestType: RequestType
+        matchedDonations: [DonationForm]
```

Comment:
i think this is good, I'm just wondering about what would happen if a donation form is "contributed" more than once to the same request - would we just append the contribution tuple to the request's `matchedDonation` array? or would we squash the tuples so that each tuple is uniquely identified by the `donationForm` id (i.e. no two `DonationFormContributionTuple` in the `matchedDonations` array would have the same `donationForm` id field)